src/gas-receiver/AxelarGasReceiverProxy.sol

```diff
@@ -0,0 +1,58 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.9;
+
+import { AxelarGasReceiver } from './AxelarGasReceiver.sol';
+
+contract AxelarGasReceiverProxy {
```

Comment:
With this proxy pattern, we cannot possibly upgrade the state layout in the implementation at all. Please consider one of those patterns from https://eip2535diamonds.substack.com/p/understanding-delegatecall-and-how?s=r.

Reply:
Will fix this, you are right.